solutions/array basics/max-by-now.js

```diff
@@ -0,0 +1,19 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arraySize = parseInt(readLine());
+let maximum = Number.MIN_VALUE;
```

Comment:
@DevratAgarwal  your code is failing for 1 testcase, please update line no 14 to `let maximum = Number.NEGATIVE_INFINITY;`,